### PR TITLE
Create 2015-02-06-about-Pandurang-Kolekar.md

### PR DESCRIPTION
**Pull request by pandurang-kolekar**
Created bio for Pandurang Kolekar

### CODE REVIEW

**Comment by gvwilson** [comment]
HI @pandurang-kolekar,
In order for pages to be published correctly, they need to have a header - please have a look at `2015-02-01-about-greg-wilson.md` for an example, and copy that model.
Thanks,
Greg

**Comment by pandurang-kolekar** [comment]
Hi @gvwilson,
Please check the updated bio.
Thanks,

**Comment by gvwilson** [comment]
Hi,
It looks like you've made two commits with zero additions and zero 
deletions - when I look at 
https://github.com/swcarpentry/training-course/pull/25/files, the file 
still doesn't have the header that it needs.  Can you please describe 
what you did?
Thanks,
Greg

On 2015-02-09 12:55 PM, Pandurang Kolekar wrote:

> Hi @gvwilson https://github.com/gvwilson,
> Please check the updated bio.
> Thanks,
> 
> —
> Reply to this email directly or view it on GitHub 
> https://github.com/swcarpentry/training-course/pull/25#issuecomment-73555866.

## 

Dr. Greg Wilson    | gvwilson@software-carpentry.org
Software Carpentry | http://software-carpentry.org

**Comment by pandurang-kolekar** [comment]
Hi @gvwilson ,

Sorry, I had misunderstood the header with file name. Now I have added header to my bio.
How does one add permalink? At this moment, I have just followed the header format in your bio.

Thanks,

File: _submissions/round-12/0/2015-02-10-about-pandurang-kolekar.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+---
+date: 2014-02-10
```
Comment (gvwilson) [inline _submissions/round-12/0/2015-02-10-about-pandurang-kolekar.md]: Please change the date to 2015-02-10 (this year).

File: _submissions/round-12/0/2015-02-10-about-pandurang-kolekar.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+round: Round 12
+title: About Pandurang Kolekar
+author: Pandurang Kolekar
+permalink: /2015/02/pandurang-kolekar/
+tags:
+  - Biography
+---
+
+#Pandurang Kolekar
```
Comment (gvwilson) [inline _submissions/round-12/0/2015-02-10-about-pandurang-kolekar.md]: Please remove this line - the title will appear automatically based on the 'title' field in the header.

**Comment by pandurang-kolekar** [comment]
You mean line no. 11?
+#Pandurang Kolekar

**Comment by gvwilson** [comment]
Yes - that line creates a level-1 heading (h1), but the template used to generate the page does that automatically, so that line should be deleted.

**Comment by pandurang-kolekar** [comment]
I hope its fine now. Please check!
Is there any documentation to create such .md files? That will help to read further about it.

**Pull request merged**